rtlib/include/rt/LandmarkDetector.hpp

```diff
@@ -0,0 +1,22 @@
+
+#pragma once
+
+#include <opencv2/core.hpp>
+
+namespace rt
+{
+    class LandmarkDetector {
+    public:
+        using LandmarkPair = std::pair<cv::Point2f, cv::Point2f>;
+        std::vector<LandmarkPair> compute(int num_matches);
+        std::vector<LandmarkPair> getTransform() {return output;}
+        // check on copying image Mat syntax
+        void setFixedImage(cv::Mat img) {fixedImg = img;}
+        void setMovingImage(cv::Mat img) {movingImg = img;}
+
+    private:
+        cv::Mat fixedImg;
+        cv::Mat movingImg;
+        std::vector<LandmarkPair> output;
```

Comment:
Private member variables are written in `lowerCamelCase_` with a trailing underscore, so rename:
```
fixedImg_
movingImg_
output_
```